draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -627,10 +627,34 @@ in cache before performing any followup queries.
 With these optimizations in place, and conforming DNS servers,
 using SVCB does not add network latency to connection setup.
 
+Moreover, if an A or AAAA response arrives before an SVCB response, the
+client SHOULD wait up to CD milliseconds before connecting as if the
+SVCB query returned NODATA, but MUST NOT transmit any information that
+could be altered by an SVCB response until the SVCB response arrives.
+For example, a TLS ClientHello may be altered by the "esnikeys" value
+of an SVCB response {{svcparamkeys-esnikeys}}. CD (Connection Delay)
+is a configurable parameter. The recommended value is 50 milliseconds,
+as per the guidance in {{!HappyEyeballsV2=RFC8305}}.
+
+An SVCB record is consistent with an active or in-progress connection C
+if the A or AAAA address for its SvcDomainName matches the destination
+address of C. If an SVCB record is consistent with an active or in-progress
+connection C, the client may continue using C with any information provided
+by the SVCB record. For example, if an SVCB record with a "esnikeys" value and
+"ipv4hint" or "ipv6hint" values {{svcparamkeys-iphints}} arrives, and one of
+those hints matches the address of an in-progress connection TCP connection,
+TLS should proceed on that connection using the provided ESNI keying material.
+
+If the SVCB record is not consistent with any active or in-progress connection,
```

Comment:
I don't think this paragraph is necessary.  The relevant requirements are already laid out in `{client-behavior}` and `{svcparamkeys-iphints}`.

Reply:
As mentioned above, I don't think `{svcparamkeys-iphints}` handles this case. (It handles the opposite one, namely, where hints are present.) I think it's important to include this case (paragraph) to be complete.